internal/protocoltest/restxml/api_client.go

```diff
@@ -190,10 +190,10 @@ func New(options Options, optFns ...func(*Options)) *Client {
 
 	resolveEndpointResolverV2(&options)
 
-	resolveTracerProvider(&options)
-
 	resolveMeterProvider(&options)
 
+	resolveTracerProvider(&options)
```

Comment:
Was getting crazy by this change being made whenever I regenerated my changes :P